test/unit/js/pauseResume.test.js

```diff
@@ -10,13 +10,46 @@ QUnit.config.reorder = false;
 var hooks = {
 	beforeEach: function() {
 		this.$el = $("#box1");
+		this.$el.each(function() {
```

Comment:
@netil Yes, you right! 
But as you know,  we should handle element as array item no matter how many it has.

So in any case we need to access that element using array style. 
But I didn't want to access an element by `[0]` like below. It looks hard-corded.

```js
  delete $el[0].__aniProps;
```